src/polygon.jl

```diff
@@ -0,0 +1,508 @@
+# Bram Spanoghe
+
+module polygon
+
+using Images, Colors, Plots, Random
+import Base: in
+
+export triangleevolution, Triangle, drawtriangle, colordiffsum, samesideofline, checktriangle, getboundaries, generatetriangle, triangletournament, generatepopulation, Shape, drawimage, makechildpopulation, mutatetriangle, mutatepopulation
+# It was mentioned we should only export the functions useful for the user, but on the other hand I can only test the functions I export here
+# So currently I'm exporting everything (so I can test them) even though only triangleevolution is necessary for the user.
+# Any more elegant way to handle this?
+
+"""
+A type Shape so the code can be made to work with e.g. rectangles as well. For now, only triangles are supported.
+"""
+abstract type Shape end
+
+"""
+    Triangle <: Shape
+
+I define triangles with the coordinates of their 3 points and a color.
```

Comment:
Try avoiding comments if they don't add new information. The code for `Triangle` is already crystal clear. But If you adjust the definition, there's absolutely no need for the comment:

```
struct Triangle <: Shape
     point1::ComplexF64 
     point2::ComplexF64
     point3::ComplexF64
     color::RGB{Float64}
 end
```